src/main/kotlin/com/openlattice/organizations/OrganizationDatabase.kt

```diff
@@ -0,0 +1,6 @@
+package com.openlattice.organizations
+
+data class OrganizationDatabase(
+        val oid: Int,
```

Comment:
Does this have to be `oid` or can it be something more semantically meaningful?

Reply:
https://www.postgresql.org/docs/12/datatype-oid.html

Reply:
"oid" is a postgres thing and has semantic meaning in that capacity -- it's the field  postgres uses to track unique objects internally, so tracking oids allows us to map back onto specific postgres objects across renames, etc. We could rename the val in the data class to "postgresId" or something like that, but personally, I find that way more confusing since "oid" in meaningful in the context that it's used, and if we changed it then we'd have to track an additional label that refers to a named field, vs just using the field name

Reply:
Totally makes sense. I just saw this in `api` and thought it might make it to the frontend, where `oid` is more mysterious.